gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/hive/HiveDatasetTest.java

```diff
@@ -162,5 +164,6 @@ public void testResolveConfig() throws IOException {
         "resPrefix_logicalDb_resPostfix", "Logical DB not resolved correctly");
     Assert.assertEquals(resolvedConfig.getString(HiveDatasetVersionCleaner.REPLACEMENT_HIVE_TABLE_NAME_KEY),
         "resPrefix_logicalTable_resPostfix", "Logical Table not resolved correctly");
+    Assert.assertEquals(resolvedConfig.getString(DUMMY_CONFIG_KEY_WITH_ROOT_VALUE),"testRoot");
```

Comment:
Is this the case that will be failed without this change?

Reply:
Yes. if the config key ends with the .ROOT_VALUE, it will through RuntimeException in ConfigUtils.propertiesToConfig()

Reply:
OK I got it.  Can you use ConfigUtils.STRIP_SUFFIX to replace raw string "ROOT_VALUE" here, where there's a clear explanation on why this suffix is added initially.